lib/ts/stacks.ts

```diff
@@ -0,0 +1,48 @@
+namespace Stacks {
+    export const application = Stimulus.Application.start();
+    export var _initializing = true;
+    
+    export class StacksController extends Stimulus.Controller {
+        protected getElementData(element: Element, key: string) {
+            return element.getAttribute("data-" + this.identifier + "-" + key);
+        };
+        protected setElementData(element: Element, key: string, value: string) {
+            element.setAttribute("data-" + this.identifier + "-" + key, value);
+        };
+        protected removeElementData(element: Element, key: string) {
+            element.removeAttribute("data-" + this.identifier + "-" + key);
+        };
+        protected triggerEvent<T>(eventName: string, detail?: T, optionalElement?: Element) {
+            const namespacedName = this.identifier + ":" + eventName;
+            var event : CustomEvent<T>;
+            try {
+                event = new CustomEvent(namespacedName, {bubbles: true, detail: detail});
+            } catch (ex) {
+                // Internet Explorer
+                event = document.createEvent("CustomEvent");
+                event.initCustomEvent(namespacedName, true, true, detail!);
+            }
+            (optionalElement || this.element).dispatchEvent(event);
+        };
+    }
+    
+    
+    type ControllersDictionary = { [identifier: string]: Stimulus.ControllerConstructor };
+    const _controllers : ControllersDictionary = {};
+    export const controllers : Readonly<ControllersDictionary> = _controllers;
```

Comment:
@bnickel Do you know if `Stacks.controllers` is used? Suggest dropping, it's available as `Stacks.application.controllers` anyway

Reply:
I think we can remove it.  Not sure what it was originally included for but I don't see any references.

Reply:
21b3b56